frontend/src/pages/TrainingReports/index.js

```diff
@@ -1,28 +1,153 @@
-import React from 'react';
+import React, { useContext, useState, useEffect } from 'react';
 import PropTypes from 'prop-types';
+import ReactRouterPropTypes from 'react-router-prop-types';
 import { Helmet } from 'react-helmet';
+import { Link, useHistory } from 'react-router-dom';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faTimesCircle } from '@fortawesome/free-solid-svg-icons';
 import './index.scss';
-// import UserContext from '../../UserContext';
+import {
+  Alert, Grid, Button,
+} from '@trussworks/react-uswds';
+import { allRegionsUserHasPermissionTo } from '../../permissions';
+import UserContext from '../../UserContext';
+import colors from '../../colors';
+import WidgetContainer from '../../components/WidgetContainer';
+import Tabs from '../../components/Tabs';
+import EventCards from './components/EventCards';
+import { getEventsByStatus } from '../../fetchers/trainingReports';
+import AppLoadingContext from '../../AppLoadingContext';
+import { EVENT_STATUS } from './constants';
 
-export default function TrainingReports() {
-  // const { user } = useContext(UserContext);
+const tabValues = [
+  { key: 'Not started', value: EVENT_STATUS.NOT_STARTED },
+  { key: 'In progress', value: EVENT_STATUS.IN_PROGRESS },
+  { key: 'Completed', value: EVENT_STATUS.COMPLETE }];
+export default function TrainingReports({ match }) {
+  const { params: { status } } = match;
+  const { user } = useContext(UserContext);
+  const [error, updateError] = useState();
+  const { setIsAppLoading, setAppLoadingText } = useContext(AppLoadingContext);
+  const [displayEvents, setDisplayEvents] = useState([]);
+  useEffect(() => {
+    async function fetchEvents() {
+      setAppLoadingText('Fetching events...');
+      setIsAppLoading(true);
+      // const filterQuery = filtersToQueryString(filtersToApply);
+      try {
+        const events = await getEventsByStatus(status);
+        setDisplayEvents(events);
+        updateError('');
+      } catch (e) {
+        // eslint-disable-next-line no-console
+        console.log(e);
+        updateError('Unable to fetch events');
+      } finally {
+        setIsAppLoading(false);
+      }
+    }
+    fetchEvents();
+  }, [status, user.homeRegionId, setAppLoadingText, setIsAppLoading]);
+
+  const regions = allRegionsUserHasPermissionTo(user);
+  const defaultRegion = user.homeRegionId || regions[0] || 0;
```

Comment:
second two cases not tested

Reply:
Assuming this was added on other pages for good reason I have added a test.